### PR TITLE
Java exercises 5-7

### PR DESCRIPTION
**Pull request by sydneynguyen912**
Completed java exercises 5-7

### CODE REVIEW

**Comment by sydneynguyen912** [comment]
exercise 5: 
image

exercise 6: 
image

exercise 7: 
image

**Review by saitj99 — APPROVED** [review]

File: sydneynguyen/java_exercises/BookTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            int id = in.nextInt();
+
+            System.out.println("Enter price for book: ");
+            int price = in.nextInt();
+
+            in.nextLine();
+            System.out.println("Enter name for book: ");
+            String name = in.nextLine();
+
+            Book newBook = new Book(id, name, price);
```
Comment (madhanramu) [inline sydneynguyen/java_exercises/BookTest.java:37]: declare `Book newBook` outside if block;
initialize here  `newBook = new Book(id, name, price);`

File: sydneynguyen/java_exercises/BookTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            System.out.println("Enter name for book: ");
+            String name = in.nextLine();
+
+            System.out.println("Enter price for book: ");
+            int price = in.nextInt();
+
+            System.out.println("Enter discount for book: ");
+            int discount = in.nextInt();
+
+            SpecialEditionBook newBook = new SpecialEditionBook(id, name, price, discount);
```
Comment (madhanramu) [inline sydneynguyen/java_exercises/BookTest.java:54]: declare `Book newBook` outside if block;
initialize here  `newBook =  new SpecialEditionBook(id, name, price, discount);`

File: sydneynguyen/java_exercises/Rectangle.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    int length;
+    int breadth; 
+
+    public Rectangle(int l, int b) {
+        length = l;
+        breadth = b;
+    }
+
+    public double area() {
+        return length * breadth;
```
Comment (madhanramu) [inline sydneynguyen/java_exercises/Rectangle.java:19]: how multiplication of two ints will result in double?

File: sydneynguyen/java_exercises/Rectangle.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package sydneynguyen.java_exercises;
+
+import java.util.Scanner;
+
+/* public Write a program to create a class Rectangle with data members length and breadth. 
+Create a method area() which finds the area of the rectangle.  
+Note: Use constructors to assign values to datamembers {} */
+
+public class Rectangle {
+    int length;
```
Comment (madhanramu) [inline sydneynguyen/java_exercises/Rectangle.java:10]: change int to double

**Review by madhanramu — CHANGES_REQUESTED** [review]
Add description and sc

**Review by madhanramu — APPROVED** [review]

**Pull request merged**